changelog.md

```diff
@@ -59,6 +59,10 @@
   1-based coordinates on POSIX for correct behaviour; the Windows behaviour
   was always correct).
 
+- The options module has been mostly rewritten. Most things should work, but
+  it's no longer possible to compare two ``none`` options. Use ``isNone``
+  instead. The comparator `==` is also gone, use `==?` instead.
```

Comment:
Yeah I was thinking about it, and I'm not sure if removing `==` is the right choice. It was done to remove some confusion with the newly introduced `==?` operator to compare options and return an option. But I guess it can be added back in.